Formula/terminally-tetris.rb

```diff
@@ -0,0 +1,24 @@
+class TerminallyTetris < Formula
+  desc "Play Tetris in the Terminal"
```

Comment:
That makes sense that it should be a personal tap. Sorry I wasn't aware that was a thing. Will close PR thanks! 👍